radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/data/action/UpdateValidatorMetadataAction.java

```diff
@@ -23,12 +23,12 @@
 
 import java.util.stream.Stream;
 
-class UpdateValidatorAction implements TransactionAction {
+class UpdateValidatorMetadataAction implements TransactionAction {
 	private final ECPublicKey validatorKey;
 	private final String name;
 	private final String url;
 
-	UpdateValidatorAction(ECPublicKey validatorKey, String name, String url) {
+	UpdateValidatorMetadataAction(ECPublicKey validatorKey, String name, String url) {
 		this.validatorKey = validatorKey;
 		this.name = name;
 		this.url = url;
```

Comment:
We actually have a few more metadata attributes.

Reply:
The `UpdateValidatorMetadataAction` is converted into `UpdateValidatorMetadata` instance while building the transaction, and it lists only these two fields.

Reply:
Never mind. I confused myself by ValidatorMetadata and TokenResourceMetadata.

Looks good to me!